docs/features/note-templates.md

```diff
@@ -106,8 +108,9 @@ For example, `filepath` can be used to customize `.foam/templates/new-note.md`,
 # This will create the note in the "journal" subdirectory of the current workspace,
 # regardless of which file is the active file.
 foam_template:
-  filepath: 'journal/$FOAM_TITLE.md'
+  filepath: 'journal/$FOAM_TITLE_SLUG.md'
```

Comment:
I am ok with the slug variable, but the Foam recommended way is still the other, so I prefer to keep FOAM_TITLE in the example

Reply:
(this applies to all the other places where the FOAM_TITLE has been replaced)

Reply:
No problem, I'll change the examples back to using the `FOAM_TITLE` variable 👌🏼